testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -253,6 +253,18 @@ public void testAccessControl()
         assertAccessAllowed("SELECT name AS my_alias FROM nation", privilege("my_alias", SELECT_COLUMN));
         assertAccessAllowed("SELECT my_alias from (SELECT name AS my_alias FROM nation)", privilege("my_alias", SELECT_COLUMN));
         assertAccessDenied("SELECT name AS my_alias FROM nation", "Cannot select from columns \\[name] in table .*.nation.*", privilege("nation.name", SELECT_COLUMN));
+        assertAccessAllowed("SELECT 1 FROM mock.default.test_materialized_view");
```

Comment:
add a test also for `SELECT count(*) FROM mock.default.test_materialized_view`

Reply:
does this add anything on top of `SELECT 1`?

Reply:
I don't think so, but that assumption is based on knowledge how semantic analyzer works. It is a another way of reading no column data, so maybe it could trigger different execution path too.